src/libultra/io/controller.c

```diff
@@ -2,6 +2,14 @@
 
 UNK_TYPE4 D_80097E40 = 0;
 
+OSPifRam __osContPifRam;
+u8 __osContLastCmd;
+u8 __osMaxControllers;
+
+OSTimer __osEepromTimer;
+OSMesgQueue D_8009CF38;
+OSMesg D_8009CF50[1];
```

Comment:
Any reason this is an array?

Reply:
Given that it's used in a function where the last argument is `count`, I assumed it was an array. I may be wrong on that. It was like that before for other cases too (although again, that's not an indication of correctness).